sklearn/decomposition/tests/test_incremental_pca.py

```diff
@@ -139,14 +139,15 @@ def test_incremental_pca_validation():
     ):
         IncrementalPCA(n_components, batch_size=10).fit(X)
 
-    # Tests that n_components is also <= n_samples.
+    # Tests that n_components is also <= n_samples in first call to partial fit
     n_components = 3
     with pytest.raises(
         ValueError,
         match=(
             "n_components={} must be"
             " less or equal to the batch number of"
-            " samples {}".format(n_components, n_samples)
+            " samples {} for the first partial_fit"
+            " call.".format(n_components, n_samples)
```

Comment:
While are at it, let's use an f-string instead of `.format` for this error message.